src/utilities.hpp

```diff
@@ -421,6 +422,10 @@ void RescaleVector(const mfem::Vector& scaling, mfem::Vector& vec);
 */
 void GetElementColoring(mfem::Array<int>& colors, const mfem::SparseMatrix& el_el);
 
+void FVMeshCartesianPartition(
```

Comment:
I know this was just copied from the earlier example files, but some of these can be made const.  We might as well take care of it in this PR.